documentation/docs/p2p.mdx

```diff
@@ -42,116 +41,72 @@ deactivate Issuer
 
 Based on the diagram above the flow is as follows:
 1. Sender submits payment to its own wallet
-2. Issuer wallet gets Open Payment metadata
-3. Issuer wallet creates invoice on receivers wallet
+2. Issuer wallet creates invoice on receivers wallet
```

Comment:
How does the sender know the asset code and scale of the recipient's account beforehand? Do they also have to query their account resource to fetch this beforehand?